easybuild/easyconfigs/a/Autotools/Autotools-20170619-foss-2017b.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'Bundle'
+
+name = 'Autotools'
+version = '20170619'  # date of the most recent change
+
+homepage = 'http://autotools.io'
+
+description = """
+ This bundle collect the standard GNU build tools: Autoconf, Automake
+ and libtool
+"""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
```

Comment:
There should be no need for this easyconfig file, neither for the `Autoconf`, `Automake` and `libtool` dependencies, since these are already available for the `GCCcore/6.4.0` toolchain.

These should get picked up automatically when building something with `foss/2017b` that requires these.

Is there a particular reason you did include these?

Reply:
not really, i saw with eb --search that there were different versions of autotools with different toolchains, so i thought to make the foss 2017 version; if its not needed i will remove this 4 files